test/extended/networking/services.go

```diff
@@ -0,0 +1,79 @@
+package networking
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/test/e2e"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("[networking] services", func() {
+	f1 := e2e.NewFramework("net-isolation1")
```

Comment:
Consider varying the framework names from those used in isolation.go to simplify debugging.